util_test.go

```diff
@@ -204,14 +218,19 @@ func TestMoveDeadNodes(t *testing.T) {
 			if nodes[i].State != StateDead {
 				t.Fatalf("Bad state %d", i)
 			}
+		case 3:
+			//Recently left node should remain at 3
+			if nodes[i].State != StateLeft {
+				t.Fatalf("Bad State #{i}")
```

Comment:
```suggestion
				t.Fatalf("Bad State %d", i)
```

Reply:
If we are changing it here we should change it in the rest of the test.

Reply:
Perhaps you are using Goland with the default code folding? It compacts format strings like
```go
t.Fatalf("Bad State %d", i)
```
into
```go
t.Fatalf("Bad State #{i}")
```
But the latter is not a valid format syntax

Reply:
yes you were right ide was playing tricks on me